app/components/moderation-base/component.js

```diff
@@ -1,30 +1,56 @@
 import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
 import Component from '@ember/component';
+
+import { task } from 'ember-concurrency';
 /**
  * The base for the provider moderation page (tabs, provider name, and breadcrumbs).
  *
  * Sample usage:
  * ```handlebars
- * {{moderation-base
- *   provider=provider
- *   pendingCount=pendingCount
- * }}
+ * {{moderation-base}}
  * ```
  * @class moderation-base
  * */
 export default Component.extend({
+    theme: service(),
+    store: service(),
+
     tabs: computed('pendingCount', function() {
         return [
             {
                 nameKey: 'global.moderation',
                 route: 'preprints.provider.moderation',
                 hasCount: true,
-                count: this.get('pendingCount') || 0,
+                count: this.get('pendingCount'),
             },
             {
                 nameKey: 'global.settings',
                 route: 'preprints.provider.settings',
             },
         ];
     }),
+
+    didReceiveAttrs() {
+        this.pendingCount = this.get('theme.provider.reviewableStatusCounts.pending');
+        this.providerName = this.get('theme.provider.name');
+        this.get('fetchData').perform();
+    },
+
+    fetchData: task(function* () {
+        try {
+            const response = yield this.get('store').findRecord(
+                'preprint-provider',
+                this.get('theme.id'), {
+                    adapterOptions: {
+                        query: { related_counts: true },
+                    },
+                    reload: true,
+                },
+            );
+            this.set('pendingCount', response.get('reviewableStatusCounts.pending'));
```

Comment:
`pendingCount` needs to be updated when we get fresh counts from a moderation list page. If a preprint is submitted while this page is open, the counts in the table will be updated, but the tab above won't until the page is refreshed:
![kapture 2017-11-21 at 14 50 08](https://user-images.githubusercontent.com/6776190/33093670-98ea301e-cecb-11e7-912d-2eb5272824b6.gif)

Maybe use an action?